docs/source/fixtures.rst

```diff
@@ -47,3 +47,35 @@ Fixtures receive a provider and an array of wallets as an argument. By default,
     // fixture implementation
   });
 
+Experimental version with using dockerized vyper for compilation.
+---------------------------
+Warring! This is experimental and the api might change without major version change.
+
+This is the recommended option if you have contracts in Vyper. It is pretty easy to set up, especially if you have Docker
```

Comment:
Remove: "This is the recommended option" (it is the only option if you have Vyper)
"It is pretty easy to set up, especially if you have Docker" -> "You will need Docker installed"